gunicorn.conf.py

```diff
@@ -1,6 +1,10 @@
 """Configuration for the gunicorn WSGI server."""
 import os
 
-bind = f':{os.environ.get("PORT", 8080)}'
-loglevel = "DEBUG"
+is_dev = os.environ.get("ENV") == "dev"
+
+# See https://docs.gunicorn.org/en/stable/settings.html
+port = os.environ.get("PORT", 8080)
+loglevel = "DEBUG" if is_dev else "INFO"
+reload = is_dev
```

Comment:
I didn't realize until now that `gunicorn` could reload-on-code-change, but it can!